CHANGELOG.rst

```diff
@@ -583,7 +587,7 @@ Changed
 - [debops.docker] If the Docker host uses a local nameserver, for example
   :command:`dnsmasq` or :command:`unbound`, Docker containers might have
   misconfigured DNS nameserver in :file:`/etc/resolv.conf` pointing to
-  ``127.0.0.1``. In these cases, the :ref:`debops.docker` role will configure
+  ``127.0.0.1``. In these cases, the debops.docker role will configure
```

Comment:
You could change the reference to point to the new role. Changelog can be updated over time in cases like this. :-)